test/e2e/framework/ginkgowrapper/wrapper.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Minor thing but we should probably omit the copyright header as we did for the other files that were copied.

Reply:
> Minor thing but we should probably omit the copyright header as we did for the other files that were copied.

Except that the license requires that we preserve the copyright notices. In fact we should add to them, not remove them:

> You must cause any modified files to carry prominent notices
          stating that You changed the files

(section 4a).

Reply:
makes sense